wls-admin-service/src/main/java/de/muenchen/oss/wahllokalsystem/adminservice/service/wahllokalbenutzer/WahllokalBenutzerModel.java

```diff
@@ -0,0 +1,14 @@
+package de.muenchen.oss.wahllokalsystem.adminservice.service.wahllokalbenutzer;
+
+import de.muenchen.oss.wahllokalsystem.adminservice.service.common.WahlbezirkArtModel;
+import java.time.LocalDate;
+import java.util.List;
+
+public record WahllokalBenutzerModel(
```

Comment:
du könntest bei den datenklassen die `@builder`-annotation hinzufügen, das erleichtert das generieren von wahllokalbenutzern in den tests
 
```suggestion
import lombok.Builder;

@Builder
public record WahllokalBenutzerModel(
```